data/_ui/interfaces.txt

```diff
@@ -1431,6 +1435,16 @@ interface "map"
 	value "max zoom" 2
 	value "min zoom" -2
 
+	anchor top
+	string "route error"
```

Comment:
You may as well use a `label` with a display condition.

Reply:
Perhaps. This was closer to the original code. Doing it this way also only required one info value.